.github/workflows/nuget.yml

```
@@ -101,6 +101,50 @@ jobs:
           echo "[target.aarch64-linux-android]" >> $CargoConfigFile
           echo "linker=`"$AndroidToolchain/bin/aarch64-linux-android21-clang`"" >> $CargoConfigFile
 
+      - name: Setup Apple Rust toolchain
+        if: runner.os == 'macOS'
+        shell: pwsh
+        run: |
+          $RustupToolchains = Resolve-Path "~/.rustup/toolchains"
+          $ToolchainPrefix = Join-Path $RustupToolchains "prebuilt"
+          New-Item -ItemType Directory -Path $ToolchainPrefix | Out-Null
+
+          $DownloadPath = "/tmp/rust-download"
+          New-Item -ItemType Directory -Path $DownloadPath | Out-Null
+          Push-Location
+          Set-Location $DownloadPath
```

Did you mean `Push-Location`?
```suggestion
          Push-Location $DownloadPath
```
I see you do a `Pop-Location` later on.

---

Push-Location saves the current location on the "stack" which you can revert to later by calling Pop-Location

Push-Location (saves current location)
Set-Location (change to whatever directory)
Pop-Location (revert back to original location)

---

Oh, I overlooked the first Push-Location :+1: